lib/fastly_nsq/rake_task.rb

```diff
@@ -15,68 +16,50 @@ def initialize(*args, &task_block)
             yield(*[self, task_args].slice(0, task_block.arity))
           end
 
-          if task_args[:channel]
-            @channel = task_args[:channel]
-          end
+          @channel  ||= require_arg :channel, task_args 
+          @topics   ||= require_arg :topics, task_args
+          @listener ||= task_args[:listener]
```

Comment:
Passing topics and channels into the listener is where we started.
This moves the addition of new topics (ATM) to listen to into a cookbook change.

I assume this is an intermediary step to something more like a CLI where you inform it of configuration via flags? and do something akin to what the `sidekiq` cli does by loading a require path?